src/ucp/api/ucp.h

```diff
@@ -584,17 +584,7 @@ enum {
      * Don't interpret address as a hint: place the mapping at exactly that
      * address. The address must be a multiple of the page size.
      */
-    UCP_MEM_MAP_FIXED    = UCS_BIT(2),
-
-    /**
-     * Create an exported memory handle that could be be packed and used by
-     * peers for their local operations on a memory buffer allocated from same
-     * or another virtual memory space, but physically registered on the
-     * same network device. A peer should call @ref ucp_mem_map with the
-     * flag @ref UCP_MEM_MAP_PARAM_FIELD_EXPORTED_MEMH_BUFFER in order to
-     * import and use the memory handle.
-     */
-    UCP_MEM_MAP_EXPORT = UCS_BIT(3)
+    UCP_MEM_MAP_FIXED    = UCS_BIT(2)
```

Comment:
we didn't release this anywhere , right ?

Reply:
correct